deepspeed/ops/sparse_attention/sparse_self_attention.py

```diff
@@ -22,7 +23,8 @@ def __init__(
         # SparsityConfig parameters needs to be set accordingly
         sparsity_config=SparsityConfig(num_heads=4),
         key_padding_mask_mode='add',
-        attn_mask_mode='mul'):
+        attn_mask_mode='mul',
+        max_seq_length=2048):
```

Comment:
Could you please add docstring for the new parameter as well?

Reply:
Sure, just added.